var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -170,20 +172,28 @@ class Gromacs(CMakePackage):
 
     depends_on("plumed+mpi", when="+plumed+mpi")
     depends_on("plumed~mpi", when="+plumed~mpi")
+    depends_on("plumed@2.8.1+mpi", when="@2022.3+plumed+mpi")
+    depends_on("plumed@2.8.1~mpi", when="@2022.3+plumed~mpi")
+    depends_on("plumed@2.8.1+mpi", when="@2021.6+plumed+mpi")
+    depends_on("plumed@2.8.1~mpi", when="@2021.6+plumed~mpi")
+    depends_on("plumed@2.8.1+mpi", when="@2020.7+plumed+mpi")
+    depends_on("plumed@2.8.1~mpi", when="@2020.7+plumed~mpi")
+    depends_on("plumed@2.7.5+mpi", when="@2021.5+plumed+mpi")
+    depends_on("plumed@2.7.5~mpi", when="@2021.5+plumed~mpi")
     depends_on("plumed@2.7.3:2.8.0+mpi", when="@2021.4+plumed+mpi")
     depends_on("plumed@2.7.3:2.8.0~mpi", when="@2021.4+plumed~mpi")
     depends_on("plumed@2.7.1:2.7.2+mpi", when="@2021+plumed+mpi")
     depends_on("plumed@2.7.1:2.7.2~mpi", when="@2021+plumed~mpi")
-    depends_on("plumed@2.7.2:2.8+mpi", when="@2020.6+plumed+mpi")
-    depends_on("plumed@2.7.2:2.8~mpi", when="@2020.6+plumed~mpi")
```

Comment:
Wait, does gromacs 2020.6 not work with plumed 2.7.2 anymore? Or why was this changed here?

Reply:
That's a great question.